workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -463,6 +508,11 @@ describe('DataSetService', () => {
         storageName: mockDataSetStorageName
       });
     });
+    it('throws when an invalid dataset Id is given.', async () => {
+      await expect(dataSetService.getDataSet(mockInvalidId)).rejects.toThrow(
+        Boom.notFound(`Could not find DataSet '${mockInvalidId}'.`)
```

Comment:
I think a `rejects.toThrow` check will only compare the message that gets passed in, not the HTTP error. If you change this to another Boom error does it still pass? I think there is a `checkHTTPError()` method that gets used to verify when an HTTP error gets thrown that you may be able to use here.

Reply:
interesting, and fix coming in next push.